src/lib/containers/entity_finder/tree/TreeNode.tsx

```diff
@@ -11,25 +11,33 @@ import { EntityHeader, ProjectHeader } from '../../../utils/synapseTypes'
 import { EntityType } from '../../../utils/synapseTypes/EntityType'
 import { EntityBadge } from '../../EntityBadge'
 import { EntityTypeIcon } from '../../EntityIcon'
+import { BUNDLE_REQUEST_OBJECT } from '../EntityFinderUtils'
 
 export type RootNodeConfiguration = {
   nodeText: string
   children: (EntityHeader | ProjectHeader)[]
 }
 
-export type TreeViewNodeProps = {
+export enum NodeAppearance {
```

Comment:
The appearance prop maps to CSS classes. `SELECT` makes the tree nodes look larger and have the same interaction properties as the `DetailsView`. `BROWSE` makes the tree nodes smaller so they don't look like they impact selection.

Reply:
Browse: 

<img width="401" alt="image" src="https://user-images.githubusercontent.com/17580037/114451381-2950c380-9ba5-11eb-9df3-9b13d624c44a.png">


Select: 

<img width="617" alt="image" src="https://user-images.githubusercontent.com/17580037/114451408-3372c200-9ba5-11eb-989c-37e25e945019.png">